src/pages/UnLikedFoodPage.js

```diff
@@ -0,0 +1,82 @@
+import styled from 'styled-components';
+import Card from '../components/Card.js';
+import Navigation from '../components/Navigation.js';
+
+export default function UnLikedFoodPage() {
+  return (
+    <LikedFoodPageStyle>
+      <Header>
+        <h1>Ich futtere lieber etwas anderes als...</h1>
+      </Header>
+      <main>
+        <ListStyle
+          data-testid={'unlikedlist'}
+          aria-describedby={
```

Comment:
I assume that you will create the list from an array with map in the near future.